src/Common/ProxyConfigurationResolverProvider.h

```diff
@@ -1,5 +1,6 @@
 #pragma once
 
+#include <atomic>
```

Comment:
Should this be <memory< ? I don't see atomic being used

Reply:
It must be in `src/Common/ProxyListConfigurationResolver.h` :)